openedx/openedx-platform
# [FAL-3383] feat: New waffleflag to enable or disable new video editor flow

## Description

This works adds a new waffleflag to enable or disable the new video editor flow on the frontend-app-course-authoring.

**Internal Jira task:** [FAL-3383](https://tasks.opencraft.com/browse/FAL-3383)

## Testing instructions

- You need to use this [branch of frontend-app-course-authoring](https://github.com/open-craft/frontend-app-course-authoring/pull/1) and this [branch of frontend-lib-content-components](https://github.com/open-craft/frontend-lib-content-components/pull/8)
- Run `make studio-static` on devstack.
- On Studio, on a new Unit, create a new video block. Verify that is redirected to the video settings page (old flow)
- Creata a video to test the edit flow.
- On the Unit, click on edit. Verify that is redirected to the video settings page (old flow)
- Go to [Django admin](http://localhost:18010/admin/waffle/flag/) and create the `new_core_editors.use_video_gallery_flow` flag. Enable it.
- On the Unit, create a new video block. Verify that is redirected to the video selection gallery.
- On the Unit, edit a video. Verify that is redirected to the video settings page.

**Release notes**

- Merge after https://github.com/openedx/frontend-app-course-authoring/pull/498 and https://github.com/openedx/frontend-lib-content-components/pull/326
